app/helpers/provider/signups_helper.rb

```diff
@@ -32,10 +32,6 @@ def gmail_inbox_link(text, email)
     link_to(text, 'https://mail.google.com/mail/u/?authuser=' + email, target: '_blank', id: 'gmail-inbox')
   end
 
-  def analytics_credentials
```

Comment:
This was the only usage of `ThreeScale::Analytics.credentials` (and hence `ThreeScale::Analytics.config` that I could find. And this method doesn't seem to be used anywhere.